pure-js/mutual-tls-example/routes/index.js

```diff
@@ -0,0 +1,109 @@
+const express = require('express');
+const https = require('https');
+const fs = require('fs');
+const crypto = require('crypto');
+const base64 = require('urlsafe-base64');
+const querystring = require('querystring');
+const axios = require('axios');
+const JOSEWrapper = require('@idpartner/jose-wrapper');
+const { v4: uuidv4 } = require('uuid');
+const config = require('../config.json');
+
+const scope = 'openid offline_access email profile birthdate address';
+
+const state = crypto.randomBytes(16).toString('hex');
+const nonce = crypto.randomBytes(16).toString('hex');
+const verifier = base64.encode(crypto.randomBytes(32));
+const challenge = base64.encode(crypto.createHash('sha256').update(verifier).digest());
+
+// Load certificate and key for mutual TLS
+const clientCert = fs.readFileSync(`./certs/${config.client_id}.pem`);
+const clientKey = fs.readFileSync(`./certs/${config.client_id}.key`);
+
+const router = express.Router();
+
+router.get('/', async (_req, res, next) => {
+  try {
+    return res.render('index', { title: 'RP Mutual TLS Example using node-oidc-client', config });
+  } catch (error) {
+    return next(error);
+  }
+});
+
+router.get('/button/oauth', async (req, res, next) => {
+  try {
+    const { iss, idp_id: idpId, visitor_id: visitorId } = req.query;
+    if (iss) {
+      req.session.issuer = iss;
```

Comment:
This is not using PAR. I think we should try to do so for at least for mTLS integrations

Reply:
i added it